doc/ADMIN.md

```diff
@@ -3,9 +3,15 @@
 This package handle the migration from Garradin to Paheko. For that, you will have to upgrade your Garradin application with this repository. This can only be done from the command-line interface - e.g. through SSH. Once you're connected, you simply have to execute the following:
 
 ```bash
-sudo yunohost app upgrade garradin -u https://github.com/YunoHost-Apps/paheko_ynh --debug
+sudo yunohost app upgrade garradin -u https://github.com/YunoHost-Apps/paheko_ynh/tree/garradin-migration --debug
 ```
 
 The --debug option will let you see the full output. If you encounter any issue, please report it aand paste the logs.
 
 **Important**: After the migration, you'll have to wait a couple of minutes (at most 3 minutes) before you can start using Paheko.
+
+Once the migration is done, you should then upgrade to new release of Paheko.
+
+If you wish, you can add personal costum configurations by editing the `/var/www/paheko/config.local.user.php` file. To see the possibilities, you can read the `/var/www/paheko/config.dist.php` file first.
```

Comment:
Meh I don't know what to do with this ... this seems like useful info for garradin's users, but people already on Paheko are not going to be interested to have this info in their webadmin ...

Reply:
Ok, I have put these instructions on garradin, please add a review here https://github.com/YunoHost-Apps/garradin_ynh/pull/106
And remove from here